zebra-consensus/src/primitives/ed25519.rs

```diff
@@ -120,43 +126,32 @@ impl Verifier {
 
     /// Flush the batch using a thread pool, and return the result via the channel.
     /// This function returns a future that becomes ready when the batch is completed.
```

Comment:
Can you please update the docs of all the methods you've modified?
The return values are now different.

Reply:
Updated in https://github.com/ZcashFoundation/zebra/pull/6887/commits/02684ca1349a0c0513feecdba27a0dd5afb02a18.